libs/traits/src/lib.rs

```diff
@@ -206,31 +206,6 @@ pub trait PoolWriteOffPolicyMutate<PoolId> {
 	fn worst_case_policy() -> Self::Policy;
 }
 
-/// A trait that can be used to retrieve the current price for a currency
-pub struct CurrencyPair<CurrencyId> {
-	pub base: CurrencyId,
-	pub quote: CurrencyId,
-}
-
-pub struct PriceValue<CurrencyId, Rate, Moment> {
-	pub pair: CurrencyPair<CurrencyId>,
-	pub price: Rate,
-	pub last_updated: Moment,
-}
-
-pub trait CurrencyPrice<CurrencyId> {
-	type Rate;
-	type Moment;
-
-	/// Retrieve the latest price of `base` currency, denominated in the `quote`
-	/// currency If `quote` currency is not passed, then the default `quote`
-	/// currency is used (when possible)
-	fn get_latest(
-		base: CurrencyId,
-		quote: Option<CurrencyId>,
-	) -> Option<PriceValue<CurrencyId, Self::Rate, Self::Moment>>;
-}
-
```

Comment:
This was not used, but if it's intended to be used very soon, I can revert this. Otherwise, I would be removed.